docs/spl-docs/regex-command.md

```diff
@@ -17,7 +17,7 @@ The required syntax is in **bold**.
 
 **Syntax:** `"<string>"`\
 **Description:** \
-This argument must be an unanchored regular expression compatible with RE2 syntax and enclosed in double quotes.\
+This argument must be a regular expression compatible with RE2 syntax and enclosed in double quotes.\
```

Comment:
I think it's still worth mentioning that it's unanchored. What I meant by my previous comment on this is that `This argument must be an unanchored regular expression` indicates that the regular expression cannot be anchored, which is not true. Clarifying that it is unanchored unless you explicitly anchor it provides some useful information to the reader

Reply:
have added this info in the latest commit.
By default, the `<regex-expression>` is considered unanchored unless you explicitly anchor it.